CMakeLists.txt

```diff
@@ -26,6 +26,16 @@ include(GNUInstallDirs)
 include(FortranCInterface)
 include(CMakePackageConfigHelpers)
 
+# Requires OpenMP
```

Comment:
Sorry that comes from an earlier attempt to fix this. WHen calling from threaded code, you need to set the `recursive` flags. You can also enable openmp, which enables `recursive` on its own.

My first fix just enabled openmp, so that comment can be removed